e107_plugins/forum/shortcodes/batch/view_shortcodes.php

```diff
@@ -804,7 +804,7 @@ function sc_buttonsx()
 	}
 		$replyUrl = "<a class='btn btn-primary".($url ?"":" disabled")."' "
 			.($url?"":" data-toggle='tooltip' title='".LAN_FORUM_0046."'
-	style='cursor: not-allowed; pointer-events: all !important;'")." href='".($url ?:"#")."'>".LAN_FORUM_2006."</a>";
+	style='cursor: not-allowed; pointer-events: all !important;'")." href='".($url ?:"#")."'>".LAN_FORUM_2006."</a>".($url?"":"<span></span>");
 
```

Comment:
<span></span> isn't valid. Can you add an &nbsp; inside without messing up the layout?